### ISSUE TITLE
The calculation of logits gets stuck

### ISSUE THREAD

**Issue by lsx-buu** [om-ai-lab/VLM-R1#111]
When my code reaches the line `logits = model(input_ids, attention_mask=attention_mask, pixel_values=pixel_values, image_grid_thw=image_grid_thw).logits`, it gets stuck. The GPU memory usage is very low, but the GPU utilization reaches 100%. What could be the possible causes?

**Comment by JesseZZZZZ** [comment]
same, question, did you solve it ?
